conjur-api/Variable.cs

```diff
@@ -47,17 +48,37 @@ public string GetValue()
         /// </summary>
         /// <param name="val">Secret value.</param>
```

Comment:
Do we need to mark this method as [Obsolete](https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?redirectedfrom=MSDN&view=netframework-4.7.2), WDYT @InbalZilberman?
Or just write a doc that it's less secure than the same method with byte array.